src/assets/scripts/client/math/flightMath.js

```diff
@@ -25,27 +25,34 @@ import {
 import { PHYSICS_CONSTANTS } from '../constants/globalConstants';
 
 /**
+ * Calculate the radius of turn of the aircraft, given its groundspeed
+ * 
+ * Reference:
+ * http://www.flightlearnings.com/2009/08/26/radius-of-turn/
+ * 
+ * Using constant of 68416 (to yield nm) in lieu of 11.26 (yields ft)
+ * 
  * @function calcTurnRadius
- * @param speed {number} currentSpeed of an aircraft
- * @param bankAngle {number} bank angle of an aircraft
- * @return {number}
+ * @param speed {number} aircraft groundspeed, in knots
+ * @param bankAngle {number} bank angle to use, in radians
+ * @return {number} radius of turn, in nautical miles
  */
-export const calcTurnRadius = (speed, bankAngle) => {
-    return (speed * speed) / (PHYSICS_CONSTANTS.GRAVITATIONAL_MAGNITUDE * tan(bankAngle));
-};
+export function calcTurnRadius(speed, bankAngle) {
+    return (speed * speed) / (68416 * Math.tan(bankAngle));
```

Comment:
Does this 68416 have a name we can use for a constant?

Reply:
it's kinda just a constant. we can call it `const conversionFactor = 68416;` if you like.

Reply:
done